### PR TITLE
helm chart for acmeair

### PR DESCRIPTION
**Pull request by harche**
Signed-off-by: Harshal Patil <harshal.patil@in.ibm.com>

### CODE REVIEW

File: helm_chart/values.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+supportService:
+  name: "support-service"
+  image: "acmeair_support_service_liberty1:latest"
+  replicas: 1
+  port: 80
+
+custIngress:
+  # The host name must be DNS resolvable
+  host: "cluster-1518518865078-master.openpowercontainer.com"
```
Comment (bpradipt) [inline helm_chart/values.yaml]: This is not needed and should be left empty.. Applies to all the below 'host' key

File: helm_chart/templates/auth-deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  annotations:
+    kompose.cmd: ../kompose/kompose convert -c
```
Comment (bpradipt) [inline helm_chart/templates/auth-deployment.yaml]: these annotations are not really needed. Applies to all the yamls

File: helm_chart/templates/auth-deployment.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    kompose.version: 1.8.0 (0c0c027)
+  creationTimestamp: null
```
Comment (bpradipt) [inline helm_chart/templates/auth-deployment.yaml]: we can avoid the creationTimestamp entry as well for all the yamls

File: helm_chart/templates/auth-deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  labels:
+    io.kompose.service: {{ .Values.authService.name }}
```
Comment (bpradipt) [inline helm_chart/templates/auth-deployment.yaml]: we can simply use 'service' as label. Applies to all

File: helm_chart/templates/auth-deployment.yaml [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        - name: MONGO_DBNAME
+          value: acmeair_sessiondb
+        image: {{ .Values.authService.image }}
+        ports:
+          - containerPort: {{ .Values.customerService.port }} 
+        imagePullPolicy: {{ default "IfNotPresent" .Values.image.pullPolicy | quote }}
+        name: {{ .Values.authService.name }}
+        resources: {}
+      restartPolicy: {{ default "Always" .Values.image.restartPolicy | quote }} 
+status: {}
```
Comment (bpradipt) [inline helm_chart/templates/auth-deployment.yaml]: Better to remove the status entry from all the yamls

File: helm_chart/templates/auth-service.yaml [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+  creationTimestamp: null
+  labels:
+    io.kompose.service: {{ .Values.authService.name }} 
+  name: {{ .Values.authService.name }} 
+spec:
+  ports:
+  - port: {{ .Values.authService.port }} 
+  selector:
+    io.kompose.service: {{ .Values.authService.name }}
+status:
```
Comment (bpradipt) [inline helm_chart/templates/auth-service.yaml]: Both status and loadBalancer keys are not needed. Applies to all the yamls

File: helm_chart/values.yaml (hunk 1 above)
Comment (harche) [reply helm_chart/values.yaml]: I am able to address all other comments (and I have updated the PR accordingly) except for this one. If I set `host` to empty then I see the problem of http getting redirected to https as well as all microservices APIs returning 301 status.

**Review by bpradipt — APPROVED** [review]

**Pull request merged**